### PR TITLE
surah HeaderView integration with SurahView

### PR DESCRIPTION
**Pull request by helloseyedjafari**
https://github.com/islamversity/Reyan/projects/1#card-46265520

### CODE REVIEW

File: surah/src/main/res/layout/row_aya.xml [hunk 1]
```diff
@@ -1,11 +1,11 @@
 <?xml version="1.0" encoding="utf-8"?>
 <RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:app="http://schemas.android.com/apk/res-auto"
-    xmlns:tools="http://schemas.android.com/tools"
-    android:layout_width="match_parent"
-    android:layout_height="wrap_content"
-    android:layout_marginLeft="@dimen/large_margin"
-    android:layout_marginRight="@dimen/large_margin">
+                xmlns:app="http://schemas.android.com/apk/res-auto"
```
Comment (rezaiyan) [inline surah/src/main/res/layout/row_aya.xml:3]: Please add a code style file, to not having these unwelcomed changes!

File: surah/src/main/res/layout/row_aya.xml [hunk 2]
```diff
@@ -95,7 +96,7 @@
         android:textColor="@color/green_dark"
         android:textSize="18dp"
         tools:ignore="RtlHardcoded,SpUsage"
-        tools:text="بسم الله الرحمن الرحیم" />
+        tools:text="بسم الله الرحمن الرحیم"/>
```
Comment (rezaiyan) [inline surah/src/main/res/layout/row_aya.xml:99]: we have it in the `settings/strings.xml`

File: surah/src/main/res/layout/surah_header.xml [hunk 3]
```diff
@@ -13,20 +13,20 @@
         android:layout_marginHorizontal="@dimen/large_margin"
         android:scaleType="fitXY"
         android:src="@drawable/ic_aya_prop_rect"
```
Comment (rezaiyan) [inline surah/src/main/res/layout/surah_header.xml:15]: Isn't better to have a gradient shape of vector drawable?

Comment (rezaiyan) [reply surah/src/main/res/layout/surah_header.xml:15]: It's better not to set `fitXY` for `scaleType`

**Review by rezaiyan — CHANGES_REQUESTED** [review]
- [ ] The Toolbar has no elevation.
- [ ] The back button is not center vertical aligned.
- [ ] It seems the padding of Bismillah's view is a little low.
- [ ] The number of aya is not aligned with the first line of aya.
- [ ] The background of the number in the surah header (+surah row) is not the same as the juz row ( juz is preferred).

File: surah/src/main/res/layout/row_aya.xml (hunk 2 above)
Comment (helloseyedjafari) [reply surah/src/main/res/layout/row_aya.xml:99]: it's `tools` so does not really matter

**Comment by helloseyedjafari** [comment]
@rezaiyan 
I created a new issue for your comments regarding ui improvements
https://github.com/islamversity/Reyan/issues/55

**Pull request merged**